Project/Assets/ML-Agents/Examples/SharedAssets/Scripts/ProjectSettingsOverrides.cs

```diff
@@ -43,6 +43,8 @@ public void Awake()
             Physics.defaultSolverIterations = solverIterations;
             Physics.defaultSolverVelocityIterations = solverVelocityIterations;
 
+            // Make sure the Academy singleton is initialized first, since it will create the SideChannels.
+            var academy = Academy.Instance;
```

Comment:
are we still using Academy? I thought it was deprecated.

Reply:
I'm not super sure about this. It comes from an earlier commit from @chriselion.

Reply:
Academy was changed from a MonoBehaviour to a singleton, but it is not deprecated and will not be any time soon.

The line below (SideChannelUtils.GetSideChannel) tries to access a SideChannel that is created when the Academy is initialized. Since the Academy initialization is lazy, it's hard to guarantee that it has been done at this point (it depends on whether Agents are initialized before or after this). Thus, this line ensures that the Academy is initialized if it wasn't already, and has no effect if it already is.

Feel free to move as much as you think is relevant into the comment.